cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,101 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let worker;
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateWorker').then(generateWorker => {
+      worker = generateWorker
+    });
+  })
 
+  it('should contain the pagination', () => {
+    cy.get('.-pagination').should('exist');
+    cy.contains('.-previous > .-btn', 'Previous');
+    cy.contains('.-next > .-btn', 'Next');
+    cy.get('.-pageInfo').should('contain.text','Page')
+  });
+
+  it('should provide an ability to select the rows count', () => {
+    cy.get('[aria-label="rows per page"]').should('exist').and('contain.text', '10 rows');
+    cy.get('[aria-label="rows per page"]').select('5 rows');
+    cy.get('[aria-label="rows per page"]').select('20 rows');
+    cy.get('[aria-label="rows per page"]').select('25 rows');
+  });
+
+  it('should provide an ability to add new worker', () => {
+    cy.contains('[id="addNewRecordButton"]', 'Add').click();
+    cy.findByPlaceholder('First Name').type(worker.firstName);
+    cy.findByPlaceholder('Last Name').type(worker.lastName);
+    cy.findByPlaceholder('name@example.com').type(worker.email);
+    cy.findByPlaceholder('Age').type(worker.age);
+    cy.findByPlaceholder('Salary').type(worker.salary);
+    cy.findByPlaceholder('Department').type(worker.department);
+    cy.contains('[id="submit"]', 'Submit').click();
+    cy.get('.rt-tr').should('contain', worker.firstName)
+  });
+
+  it('should provide an ability to delete the worker', () => {
+    cy.get('#delete-record-1').click();
+    cy.get('.rt-tr').should('not.contain', worker.firstName)
+  });
+
+  it('should provide an ability to delete all workers', () => {
+    cy.get('#delete-record-1').click();
+    cy.get('#delete-record-2').click();
+    cy.get('#delete-record-3').click();
+    cy.get('.rt-tr').should('not.contain', worker.firstName)
```

Comment:
better to use loop here and you don't check that your workers have been deleted. You check not existing worker again